### PR TITLE
AP_AHRS: apply AHRS trim in EKF_TYPE_SITL

### PR DESCRIPTION
**Pull request by kd0aij**
apply AHRS_TRIM in EKF_TYPE_SITL

### CODE REVIEW

File: libraries/AP_AHRS/AP_AHRS_NavEKF.cpp [hunk 1]
```diff
@@ -312,6 +312,11 @@ void AP_AHRS_NavEKF::update_SITL(void)
 
         fdm.quaternion.rotation_matrix(_dcm_matrix);
 
+        // apply AHRS trims (this seems to be correct)
+        _dcm_matrix = _dcm_matrix * get_rotation_vehicle_body_to_autopilot_body();
+        // this has no effect on flightaxis sim
+//        (_dcm_matrix * get_rotation_vehicle_body_to_autopilot_body()).to_euler(&roll, &pitch, &yaw);
```
Comment (magicrub) [inline libraries/AP_AHRS/AP_AHRS_NavEKF.cpp]: Remove the old one

**Comment by kd0aij** [comment]
@magicrub I don't understand why the comment line doesn't work... so I'm not sure whether this is correct yet.

**Comment by tridge** [comment]
this doesn't look right, the trim affects _body_dcm_matrix, which should already be being updated in AP_AHRS_DCM::euler_angles()

**Comment by tridge** [comment]
I've updated this with what I think is the right fix. Can you test?

**Comment by kd0aij** [comment]
that makes sense; will test soon

**Comment by kd0aij** [comment]
Works for me in RF8

**Comment by kd0aij** [comment]
@tridge I think this never made it into master, but it works properly in RF8/SITL.

**Pull request merged**